monitoring/terraform/modules/slack_alert_on_cloudfront_errors/main.tf

```diff
@@ -43,5 +43,7 @@ module "slack_secrets" {
 module "lambda_error_alerts" {
   source = "../slack_alert_on_lambda_error"
 
-  account_name = var.account_name
+  # We need to add a suffix here, so this doesn't conflict with the
+  # "alert on Lambda errors" Lambda that lives in eu-west-1 in this account.
+  account_name = "${var.account_name}_useast1"
```

Comment:
Would it be appropriate to compose the account name from the account and region in all cases for consistency?

I can see why not - if this is the only one in another region, then leaving the ones in the default region unmarked is appropriate.

Reply:
I considered it, but as you guessed us-east-1 is a special case – almost all of our infra runs in eu-west-1. CloudFront is the only thing we run elsewhere.